File: 14-longest-common-prefix/14-longest-common-prefix.php
<?php

class Solution {

    /**
     * @param String[] $strs
     * @return String
     */
    
function longestCommonPrefix(array $strs) {
    if (empty($strs)) {
        return "";
    }
    if (count($strs) == 1) {
        return $strs[0];
    }
    sort($strs);
    $prefix = "";
    for ($i = 0; $i < strlen($strs[0]); $i++) {
        if ($strs[0][$i] == $strs[count($strs) - 1][$i]) {
            $prefix .= $strs[0][$i];
        } else {
            break;
        }
    }
    return $prefix;
}


}

File: 14-longest-common-prefix/14-longest-common-prefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        if (strs.Length == 0) {
            return "";
        }
        if (strs.Length == 1) {
            return strs[0];
        }
        Array.Sort(strs);
        string prefix = "";
        for (int i = 0; i < strs[0].Length; i++) {
            if (strs[0][i] == strs[strs.Length - 1][i]) {
                prefix += strs[0][i];
            } else {
                break;
            }
        }
        return prefix;
    }
}